lib/api/1.1/southbound/profiles.js

```diff
@@ -31,9 +31,13 @@ function profilesRouterFactory (
      */
 
     router.get('/profiles', function (req, res) {
+        if (req.query.mac && req.query.ip) {
```

Comment:
Right,  there are a lot of route code that reference query parameters that offer no description.